### DISCUSSION TITLE
Autograder报内部错误：运行超时

### DISCUSSION THREAD

**Discussion by tiger9tu** [arcsysu/SYsU-lang#106] [Q&A]
Autograder在第635个样例median处时产生运行了超时的内部错误，并停止了执行。对此我有两点疑问：
1.这里的运行超时是由于什么原因产生的？如果是程序运行超时的话不是应该输出Timeout然后继续执行吗？
2.我在本地的docker环境中执行测试，通过了第635个样例，并顺利执行了包括该样例在内的所有样例，请问autograder的测试和本地docker环境的测试有什么区别吗？为什么我在本地docker环境能够通过的测试，会在autograder上产生内部错误？
报错截图如下：
![image](https://github.com/arcsysu/SYsU-lang/assets/76516415/ca343f4e-37ac-4686-b81f-c38ab26581cf)

**Comment by TrueStar02** [comment]
运行的时间限制是两个小时,如果两个小时还没有运行完所有样例就会强制结束.

**Reply by xuantengh** [reply]
正确的

**Comment by xuantengh** [comment]
@Tiger9Tu 运行时长限制已改成 3h.
